README.md

```diff
@@ -131,11 +131,51 @@ Grab a coffee, the build will take a few minutes to complete.
 
 #### Add GTK to Your Environmental Variables
 
-```PowerShell
-$env:Path = "C:\gtk-build\gtk\x64\release\bin;" + $env:Path
-$env:LIB = "C:\gtk-build\gtk\x64\release\lib;" + $env:LIB
-$env:INCLUDE = "C:\gtk-build\gtk\x64\release\include;C:\gtk-build\gtk\x64\release\include\cairo;C:\gtk-build\gtk\x64\release\include\glib-2.0;C:\gtk-build\gtk\x64\release\include\gobject-introspection-1.0;C:\gtk-build\gtk\x64\release\lib\glib-2.0\include;" + $env:INCLUDE

```

Comment:
Hi @mitchhentges, thanks so much for the contribution!

Why did you remove the LIB and INCLUDE paths? At least for allowing PyGObject and pycairo to build we need them, unless there is another alternative.

Reply:
Aha, I was wondering why those were in there! I assumed that maybe there were _for_ Visual Studio, and therefore were made redundant.

Would it be OK to move these environment variable settings to the PyGObject/pycairo doc section specifically? Otherwise,  if we kept setting them in the "general" docs, perhaps we should broaden them to include _all_ the possible `INCLUDE` paths (such as `./include/pango-1.0`, `include/gdk-pixbuf-2.0`, `include/gtk-4.0`, etc). However, I'd lean a bit against this considering that the Visual Studio and Rust use cases don't need `LIB` and `INCLUDE`, for what it's worth :)

Reply:
That sounds great!

Reply:
Hmm, I'm not sure if it's needed for the `gvsbuild build --enable-gi --py-wheel gtk4 pygobject` or the `pip install ...`, so I added it before both.

I tried to reproduce the issue locally (by running the steps without `LIB` and `INCLUDE`), but instead ran into an issue  during `gvsbuild build --enable-gi --py-wheel ...` where it couldn't find the `python3` command. I tried duplicating `python.exe` into `python3.exe` in both my `gvsbuild` venv _and_ my actual Python installation (installed via the [python.org installer](https://www.python.org/downloads/windows/)), but I couldn't work around it. I tried installing `python` into MSYS2, but then it complained that `distutils` wasn't there (even though MSYS2 had Python 3.11, and `import distutils` _was_ working! _And_ my python.org, native Python was also 3.11! Weird).

I was able to work around it with `--use-env`, but then neither `LIB` nor `INCLUDE` were needed: I was able to do the build _and_ the `pip install` commands without an issue.

So, putting it at the top of the "Use PyGObject" is my safest guess 😅